lhapdf.sh

```diff
@@ -3,12 +3,22 @@ version: v6.1.5
 source: https://github.com/alisw/LHAPDF
 tag: v6.1.5
 requires:
-  - yaml-cpp
+ - yaml-cpp
+ - boost
+ - "autotools:(slc5).*"
 ---
 #!/bin/sh
 # Does not work out of source.
 
 cd $SOURCEDIR
+
+# autoreconf is needed for SLC 5
+# because we have to use built autotools
+if [[ $ARCHITECTURE =~ "slc5.*" ]]; then
```

Comment:
Why not key-ing on the existance of $AUTOTOOLS_ROOT, rather than the architecture? This way you do not need to keep the regexp in sync in two different places.